CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Splunk Enterprise SDK for Python Changelog
 
+## Version 2.0.0-beta
+
+### Feature updates
+* `ensure_binary`, `ensure_str`, `ensure_text` and `assert_regex` utility methods have been migrated from `six.py` to `splunklib/__init__.py`
+
+### Major changes
+* Removed Code specific to Python2
+* Removed six.py dependency
+* Removed `__future__` imports
+* Refactored & Updated `splunklib` and `tests` to utilise Python3 features
+* Updated CI test matrix to run with Python versions - 3.7, 3.9 and 3.10.7
+* Refactored Code throwing `deprecation` warnings
```

Comment:
Seeing some deprecation warnings in the test runs, are these expected?

Reply:
yes those warnings are expected as they are regarding couple of features that we have replaced but have to decide on when to remove them permanently from the code.